packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -212,8 +203,17 @@ module.exports = {
           name: 'static/media/[name].[hash:8].[ext]'
         }
       }
+      // ** STOP ** Are you adding a new loader?
+      // Remember to add the new extension(s) to the "url" loader exclusion list.
     ]
   },
+  // @remove-on-eject-begin
+  // Point ESLint to our predefined config.
+  eslint: {
+    configFile: path.join(__dirname, '../eslintrc'),
+    useEslintrc: false,
```

Comment:
Is it possible to use `emitWarning: true` still? Without it, linting errors fail the dev builds entirely.
![image](https://cloud.githubusercontent.com/assets/1699281/25681418/dca8697a-3019-11e7-8365-1df2493e2810.png)